rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -19,6 +19,9 @@ info:
     name: GNU Lesser General Public License
     url: 'http://www.gnu.org/licenses/lgpl.html'
 
+servers:
```

Comment:
I didn't know Maven properties would populate this file. Did we configure a maven plugin to find and replace properties in this file?

Reply:
I just copied this from the discovery-api repo.